### PR TITLE
Dynamic index

### PR DESCRIPTION
**Pull request by Eomm**
This gives to the user the possibility to customize the index date name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,27 @@ You can then use [Kibana](https://www.elastic.co/products/kibana) to
...
+
+It is possible to customize the index name at every log just providing a function to the `index` option:
+
+```js
+const pino = require('pino')
+const pinoElastic = require('pino-elasticsearch')
+
+const streamToElastic = pinoElastic({
+  index: function (logTime) {
+    // the logTime is a string in ISO time when the logs happened
```
Comment (mcollina) [inline README.md]:
```suggestion
    // the logTime is a ISO 8601 formatted string of the log line
```

File: README.md [hunk 2]
```diff
@@ -59,6 +59,27 @@ You can then use [Kibana](https://www.elastic.co/products/kibana) to
 browse and visualize your logs.  
 **Note:** This transport works only with Elasticsearch version ≥ 5.
 
+#### Dynamic index
+
+It is possible to customize the index name at every log just providing a function to the `index` option:
```
Comment (mcollina) [inline README.md]:
```suggestion
It is possible to customize the index name for every log line just providing a function to the `index` option:
```

**Comment by mcollina** [comment]
cc @delvedor @jsumners

**Comment by Eomm** [comment]
Sorry to bother you all :pray: 
May I help or improve this PR to get this feature released?

**Pull request merged**